src/prpy/planning/chomp.py

```diff
@@ -84,7 +85,20 @@ def sync(self, robot):
                         body_name, os.path.basename(cache_path)
                     )
 
-                    self.module.computedistancefield(body, cache_filename=cache_path)
+                    # Temporarily disable other bodies for distance field computation
+                    other_bodies = []
+                    for other_body in self.env.GetBodies():
+                        if other_body == body:
+                            continue
+                        if not other_body.IsEnabled():
```

Comment:
`body.IsEnabled()` returns `True` if any link is enabled [0] so this check does the right thing, but I removed the check for simplicity.

[0] http://openrave.org/docs/latest_stable/coreapihtml/classOpenRAVE_1_1KinBody.html#a29a5ffa80c81125b8d5ccb40e4fe44be